projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -38,57 +49,106 @@ const PieSemiCircleChart: FC< PieSemiCircleChartProps > = ( {
 	height,
 	label,
 	note,
+	showTooltips = false,
 } ) => {
 	const providerTheme = useChartTheme();
+	const [ tooltipData, setTooltipData ] = useState< TooltipData | null >( null );
 	const centerX = width / 2;
 	const centerY = height;
+	const radius = Math.min( width, height ) / 3;
```

Comment:
Why is it divided by 3?

Reply:
The division by 3 in const radius = Math.min(width, height) / 3 helps create a balanced layout. We take the smaller dimension with Math.min() to prevent overflow, then divide by 3 to make the pie chart take up about a third of the space. This leaves room for labels while keeping the chart visible and well-proportioned. We could use other values (like /2 for a bigger pie or /4 for a smaller one)- the layout and location of the label and note, and general layout of this particular chart has been a bit of a pain so far! Happy to experiment with other dimensions?